src/main/java/uk/gov/hmcts/reform/hmc/service/HearingManagementServiceImpl.java

```diff
@@ -356,13 +356,28 @@ private void validateHearingRequestDetails(UpdateHearingRequest hearingRequest)
     }
 
     private void validateHearingDetails(HearingDetails hearingDetails) {
+        validateHearingWindow(hearingDetails);
+
+        if (hearingDetails.getDuration() % 5 != 0) {
+            throw new BadRequestException(INVALID_DURATION_DETAILS);
+        }
+    }
+
+    private void validateHearingWindow(HearingDetails hearingDetails) {
         if (hearingDetails.getHearingWindow().getDateRangeEnd() == null
             && hearingDetails.getHearingWindow().getDateRangeStart() == null
             && hearingDetails.getHearingWindow().getFirstDateTimeMustBe() == null) {
             throw new BadRequestException(INVALID_HEARING_WINDOW);
         }
-        if (hearingDetails.getDuration() % 5 != 0) {
-            throw new BadRequestException(INVALID_DURATION_DETAILS);
+        if ((hearingDetails.getHearingWindow().getDateRangeEnd() != null
```

Comment:
Can the error be changed to Hearing window details are invalid instead of INVALID_HEARING_WINDOW as details are present but not in the expected behaviour.

Reply:
Not many references to INVALID_HEARING_WINDOW so I changed the message to what you recommended and added a new variable, HEARING_WINDOW_EMPTY_NULL, which has the previous message